crates/commands/tests/tests-init.rs

```diff
@@ -129,7 +129,7 @@ out = "out"
 libs = ["dependencies"]
 
 [dependencies]
-forge-std = "1.9.4"
+forge-std = "1.9.5"
```

Comment:
This test will be made dynamic in my next PR, sorry for the unrelated breakage ^^